content/rust/concepts/functions/functions.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Functions'
+Description: 'Functions are the block of reusable code that can perform similar related actions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Game Development'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'Computer Science'
+  - 'rust-for-programmers'
+---
+
+**Rust functions** are declared using the `fn` keyword. Its arguments are type annotated, just like variables, and, if the function returns a value, the return type must be specified after an arrow `->`.
```

Comment:
The entry should start with an expanded version of the description used in the metadata above. Then there should be a ## Syntax header. All of the content on lines 15 and 16 can follow the header. Generally, we also include a pseudo code block with the general syntax laid out.

It may be helpful to review an existing concept entry-
https://www.codecademy.com/resources/docs/cpp/functions